### DISCUSSION TITLE
List of "manually closed roads"

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#18624] [Q&A] [answered]
While navigation I can select a road and mark this as "dont use" (temporarily closes)
BUT where can I find a list of all these manually edits??

**Comment by DmitryAlexei** [comment, marked as answer]
Please see https://osmand.net/docs/user/navigation/routing/osmand-routing#avoid-roads-menu

**Reply by ThomasMeister123** [reply]
Thank you. Sometimes the solution is so near ;)
